people/stevekirk.json

```diff
@@ -2,47 +2,73 @@
   "firstname": "Steve",
   "lastname": "Kirk",
   "handle": "Lockheed",
-  "birth": "",
-  "death": "2019",
-  "obituary": "",
+  "birth": "1970-08-24",
+  "death": "2019-08-05",
+  "obituary": "https://www.tributearchive.com/obituaries/7093841/steven-kirk",
   "issue": "60",
-  "mainimage": "/images/Steve_Kirk.jpg",
-  "maintext": "He was one of only a few folks who has ever lead the DEFCON NOC, and was the COO of DEFCON for a term before Speedrussr took over.",
+  "affiliations": "DEF CON, DEF CON NOC, DEF CON 101, Sony PlayStation",
+  "mainimage": "/images/stevekirk00.jpg",
+  "maintext": "He was one of only a few folks who has ever lead the DEF CON NOC, and was the COO of DEF CON for a term before Speedrussr took over.",
   "socialmedialinks": [
     {
       "sitename": "Twitter",
       "siteurl": "https://www.twitter.com/TheLockheed"
     },
+    {
+      "sitename": "Facebook",
+      "siteurl": "https://www.facebook.com/sbkirk"
+    },
+    {
+      "sitename": "Instagram",
+      "siteurl": "https://www.instagram.com/Steve_of_Clan_Kirk"
+    },
+    {
+      "sitename": "Keybase",
+      "siteurl": "https://keybase.io/lockheed"
+    },
+    {
+      "sitename": "Infocon Database",
+      "siteurl": "https://infocondb.org/presenter/lockheed"
+    },
     {
       "sitename": "Website",
-      "siteurl": "http://www.random-ether.net"
+      "siteurl": "https://web.archive.org/web/20211206104917/https://random-ether.net/"
+    },
+    {
+      "sitename": "Saber Martial Arts Foundation",
+      "siteurl": "https://sabermartialarts.life/our-force-ghost/"
     }
   ],
   "contributions": [
     {
       "title": "DEF CON NOC",
       "url": "https://www.defconnetworking.org/",
```

Comment:
This website has moved to https://noc.defcon.org as of a little while ago

Reply:
URI updated; apologies for not validating everything. Changes made and pushed upstream!